client/src/Components/App/index.css

```
@@ -8,7 +8,11 @@ html {
   font-size: 16px;
   font-family: "Roboto", sans-serif;
 }
-
-body {
+.root{
+  height: 100vh;
+}
+body ,.container {
   background-color: #f5f6f8;
+  min-height: calc(100vh - (3.59px + 70px))
```

just for the container class
```suggestion
  min-height: calc(100vh - (3.59px + 70px))
```